panda_grasp:
  task_frame_id: task
  base_frame_id: base_link_rm
  ee_frame_id: ee_link
  ee_grasp_offset:  [0.0, 0.0, -0.382, 1, 0.0, 0.0, 0]
  scan_joints:
    - [0.0, 0.0, -1.5707, 0, -1.5707, 3.1415]
    - [0.715, -0.2443, -1.553, -0.4014, -1.53588, 3.3335]
    - [-0.4014, -1.0646, -0.244, 0.17453, -2.147, 3.335 ]
    - [-1.11701, -0.593, -1.0646, 0.471, -1.7627, 4.014]
    - [-0.4712, 0.977, -2.129, 0.017, -1.5184, 3.000]

vgn_server:
  frame_id: task
  base_frame_id: base_link_rm
  model: $(find vgn)/assets/models/vgn_conv.pth
  camera:
    frame_id: camera_color_optical_frame
    info_topic: /camera/aligned_depth_to_color/camera_info
    depth_topic: /camera/aligned_depth_to_color/image_raw
  length: 0.08 # 体素格子大小
  resolution: 40
  depth_scaling: 0.001  # Set to 1.0 for Gazebo, 0.001 for a RealSense